### PR TITLE
minor refactorings of recently touched files

### PR DESCRIPTION
**Pull request by adamralph**
- DelegateProxyGenerator
 - AssignsOutAndRefParametersSpecs
 - DelegateProxyGeneratorTests

### CODE REVIEW

File: src/FakeItEasy/Creation/DelegateProxies/DelegateProxyGenerator.cs [hunk 1]
```diff
@@ -61,58 +61,53 @@ public virtual bool MethodCanBeInterceptedOnInstance(MethodInfo method, object c
...
             return Expression.Lambda(typeOfProxy, body, parameterExpressions).Compile();
         }
 
+        // Generate a method that:
+        //     - wraps its arguments in an object array
+        //     - passes this array to eventRaiser.Raise()
+        //     - assigns the output values back to the ref/out parameters
+        //     - casts and returns the result of eventRaiser.Raise()
+        //
+        //     For instance, for a delegate like this:
```
Comment (thomaslevesque) [inline src/FakeItEasy/Creation/DelegateProxies/DelegateProxyGenerator.cs]: Nitpicking: wrong indentation

Comment (adamralph) [inline src/FakeItEasy/Creation/DelegateProxies/DelegateProxyGenerator.cs]: oops!

**Comment by thomaslevesque** [comment]
LGTM, besides the minor remark about indentation.

**Comment by adamralph** [comment]
Amended the commit to fix the indentation.

**Pull request merged**